website/docs/tutorials/quick-start/index.mdx

```diff
@@ -38,7 +41,41 @@ The doctor subcommand will attempt to detect all the components and report any p
 ```bash
 meta doctor
 ```
-    TODO: Add result image/log
+After Sucessful installation, the above command produces an output somewhat similar to the one below.
+```console
+user@first-project:~$ meta doctor
+——————————————————————————— Global ———————————————————————————
+curr. directory      /Users/user/Documents/metatype-playground/projects/first-project
+global config        /Users/user/Library/Application Support/dev.metatype.meta/config.json
+meta-cli version     0.3.6
```

Comment:
Could we link this to the published version in dev/lock.yml?

Reply:
Ok, but I did `meta upgrade` and `0.3.6` was the latest one.